sdk/azidentity/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Instances of `ChainedTokenCredential` will now skip looping through the list of source credentials and re-use the first successful credential on subsequent calls to `GetToken`.
```

Comment:
The docs for `NewChainedTokenCredential` should be updated to describe this behavior.

Reply:
Good point!